snapshots/snapshots/src/main/kotlin/com/emergetools/snapshots/SnapshotImageMetadata.kt

```diff
@@ -11,6 +11,11 @@ enum class SnapshotType {
   ACTIVITY,
 }
 
+enum class SnapshotErrorType {
+  EMPTY_SNAPSHOT,
```

Comment:
Just want to flag some of the stuff @noahsmartin has been touching in this area too.